### DISCUSSION TITLE
Printing out circuits from Digital

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#1009] [Q&A] [answered]
A nice addition to DIgital would be a print function. Could this be added in a future upgrade ?

**Comment by hneemann** [comment, marked as answer]
Do you mean printing circuits?
You can export circuits to PNG or SVG and then print them with the program of your choice. 
Direct printing is a very rare use case in my opinion.  Usually the circuits are embedded in other documents and then printed.

**Comment by jongena** [comment]
Yes, I did mean to print circuits. Your answer is a nice solution: I can simply open .svg files in the browser.
and easily print from the browser. It also the question about finding names in a complex and large
circuit. I can simply search for the names in the .svg files using the find function of the browser.
